app/frontend/src/components/form/RequestReceipt.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+    <v-btn
+      color="primary"
+      class="mx-5 mb-10"
+      :disabled="success"
+      fab
+      large
+      v-on:click="displayDialog()"
+    >
+      <v-icon v-if="success">check</v-icon>
+      <v-icon v-else>email</v-icon>
+    </v-btn>
+
+    <BaseDialog
+      :show="showDialog"
+      type="CONTINUE"
+      @close-dialog="showDialog = false"
+      @continue-dialog="requestReceipt()"
+    >
+      <template v-slot:icon>
+        <v-icon large color="primary">email</v-icon>
+      </template>
+      <template v-slot:text>
+        <label>Send to E-mail Address</label>
+        <v-text-field
+          dense
+          flat
+          outlined
+          solo
+          :rules="emailRules"
+          prepend-inner-icon="email"
+          v-model="to"
+        />
+      </template>
+      <template v-slot:button-text-continue>
+        <span>SEND</span>
+      </template>
+    </BaseDialog>
+
+    <BaseDialog :show="resultDialog" @close-dialog="resultDialog = false">
+      <template v-slot:icon>
+        <v-icon v-if="success" large color="success">check_circle_outline</v-icon>
+        <v-icon v-else large color="error">cancel</v-icon>
+      </template>
+      <template v-slot:text>{{ resultDialogMsg }}</template>
+    </BaseDialog>
+  </div>
+</template>
+
+<script>
+import validator from 'validator';
+
+import emailService from '@/services/emailService';
+
+export default {
+  name: 'RequestReceipt',
+  data: () => ({
+    emailRules: [
+      v => !!v || 'E-mail is required',
+      v =>
+        validator.isEmail(v, { allow_display_name: true }) ||
+        'invalid e-mail format',
+      v => (v && v.length <= 100) || 'E-mail must be less than 100? characters'
```

Comment:
This one probably isn't relevant since we're not storing this email so there wouldn't be a limit I think (depends on CHES)
but it reminds me that I never actually set proper DB length backed limits on the UI 😬 
So a todo on that for Step2...

Reply:
CHES doesn't have an upper-bound limit on this so I can get rid of this extra check.